Dshop/apps/products_catalogue/views.py

```diff
@@ -1,3 +1,45 @@
-#from django.shortcuts import render
+from lxml import etree
 
-# Create your views here.
+from django.http import HttpResponse
+from django.views.generic import ListView
+
+from .models import Product
+
+
+class ProductListView(ListView):
+    model = Product
```

Comment:
It will not work properly because it will list all products, even inactive ones. Please use get_queryset with is active.

Reply:
Done.

Reply:
Nice!